### PR TITLE
Release 7.6.2 Add Logo Caption to IdP cards on IdP selection page

### PR DESCRIPTION
**Pull request by Baggerone**
Related to 
https://trello.com/c/Cx6VKM4B/494-use-new-logos-in-idps

### CODE REVIEW

File: themes/material/default/selectidp-links.php [hunk 1]
```diff
@@ -81,10 +81,14 @@ function clickedAnyway(idpName) {
...
-                             src="<?= empty($idp['logoURL']) ? 'default-logo.png'
-                                                             : $idp['logoURL'] ?>">
+                    <button class="mdl-button logo-container fill-parent" onclick="setSelectedIdp('<?= $idpId ?>')">
+                        <div class="image-wrapper">
+                            <img class="logo" id="<?= $idpId ?>"
+                                src="<?= empty($idp['logoURL']) ? 'default-logo.png'
+                                                                : $idp['logoURL'] ?>">
+                        </div>
+
+                        <span class="mdl-color-text--grey-600 logo-caption clickable-caption"><?= empty($idp['logoCaption']) ? '<br>': $idp['logoCaption'] ?> </span>
```
Comment (forevermatt) [inline themes/material/default/selectidp-links.php:91]: No need to block this release for this issue, but I missed this echo of the IDP logo caption earlier. It should also use `htmlentities()` like we're now doing below.

**Review by forevermatt — APPROVED** [review]

**Review by hobbitronics — APPROVED** [review]

**Comment by hobbitronics** [comment]
I added the missing `htmlentities` @forevermatt

**Comment by Schparky** [comment]
I'm surprised this is a public repo with all the org-specific details in the metadata. Probably too late to do anything about that now, though.

**Comment by Baggerone** [comment]
> I'm surprised this is a public repo with all the org-specific details in the metadata. Probably too late to do anything about that now, though.

That's just for local development, plus anyone can see the same thing by browsing to one of our apps that go through the IdP hub.

**Pull request merged**